specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5514,10 +5514,8 @@
         "adult",
         "brands",
         "categories",
-        "color",
```

Comment:
@jennifermarsman I thought the whole type was going away -- or did I misunderstand the plan?

Reply:
Nevermind, I read more closely. This looks fine.